ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/validation/ResourceAnnotationValidationsSpec.scala

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.server.api.validation
```

Comment:
Why is this test living so far from the class that it tests?

Reply:
Originally I had the tested class near this test. Then I moved the tested class because I needed to use it in in-memory stores.
I didn't move the test because there's no test target defined there in bazel yet. Though I could define it, but I think it'd be best to create a dedicated bazel package for store level api + in-memory implementations e.g. 'ledger/participant-store'